virtual_ecosystem/models/plants/plants_model.py

```diff
@@ -517,6 +531,53 @@ def _setup(
     def spinup(self) -> None:
         """Placeholder function to spin up the plants model."""
 
+    def reset_update_vars(self) -> None:
+        """Define variables used by the plants model during update."""
+
+        # Initialize variables that hold one value per cell
+        cell_template = xr.full_like(self.data["elevation"], 0)
+
+        self.data["leaf_turnover"] = cell_template.copy()
+        self.data["leaf_turnover_c_n_ratio"] = cell_template.copy()
+        self.data["leaf_turnover_c_p_ratio"] = cell_template.copy()
+        self.data["leaf_turnover_n_mass"] = cell_template.copy()
+        self.data["leaf_turnover_p_mass"] = cell_template.copy()
```

Comment:
I think this function would be cleaner using:

```suggestion
reset_vars = [
    "leaf_turnover", 
    "leaf_turnover_c_n_ratio", 
    "leaf_turnover_c_p_ratio", 
    "leaf_turnover_n_mass",
    "leaf_turnover_p_mass", 
    ...
]
for var in reset_vars:
                self.data[var] = cell_template.copy()
```